cpu/sam0_common/periph/pwm.c

```diff
@@ -135,6 +178,94 @@ static void poweroff(pwm_t dev)
 #endif
 }
 
+static void _tc_init(Tc *tc, pwm_mode_t mode, uint8_t prescaler, uint8_t res)
+{
+    /* reset TC module */
+    tc->COUNT8.CTRLA.bit.SWRST = 1;
+    while (tc->COUNT8.CTRLA.bit.SWRST) {}
+
+    /* set PWM mode */
+    switch (mode) {
+        case PWM_LEFT:
+            tc->COUNT8.CTRLBCLR.reg = TC_CTRLBCLR_DIR;     /* count up */
+            break;
+        case PWM_RIGHT:
+            tc->COUNT8.CTRLBSET.reg = TC_CTRLBSET_DIR;     /* count down */
+            break;
+        case PWM_CENTER:        /* currently not supported */
```

Comment:
Could you add a debug statement here please ?

Reply:
Or an `assert(0)`?